### PR TITLE
New Notification Toast

### PR DESCRIPTION
**Pull request by Ishimwe7**
### What does this PR do?

- This PR implements real-time notifications for users, ensuring that toast notifications are displayed whenever new notifications are received. It enhances the user experience by providing immediate feedback on new notifications without requiring a page refresh or manual data refetch.

### How should this be manually tested?

1. Run the Application: Start the application and ensure that it is connected to the WebSocket or real-time service.
2. Trigger New Notifications: Use the application or an external tool to simulate the arrival of new notifications.
3.Verify Toast Notifications:
    - Check that a toast notification appears for each new notification received.
    - Ensure that the toast notifications are styled according to the custom design.
    - Confirm that no duplicate toasts are shown for the same notification.
    - Refresh and Reconnect: Reload the page or disconnect/reconnect to the real-time service to ensure that notifications are 
    correctly handled across sessions.

### What are the relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187570833

### CODE REVIEW

File: src/contexts/notificationContext.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
+// import React, { createContext, useContext, useEffect, useRef } from 'react';
+// import { useSelector } from 'react-redux';
+// import { RootState } from '../store';
+// import { notify } from '../utils/notifyUsers';
+// import { useGetNotificationsQuery } from '../slices/notificationSlice/notificationApiSlice';
+
```
Comment (robsdagreat) [inline src/contexts/notificationContext.tsx]: Too many comments! Consider deleting them if they don't serve any purpose.

File: src/pages/seller/NotificationBar.tsx [hunk 2]
```diff
@@ -1,13 +1,74 @@
...
+// import { useGetNotificationsQuery } from "../../slices/notificationSlice/notificationApiSlice";
+// import Notification from "../../Components/seller/Notification";
+// import { formatDistanceToNow } from "date-fns";
+
+// //@ts-ignore
+// const NotificationBar: React.FC = ({ onClose }) => {
+//   const dispatch = useDispatch();
+//   //@ts-ignore
+//   const {
+//     data: sellerAllNotifications,
```
Comment (robsdagreat) [inline src/pages/seller/NotificationBar.tsx]: same here

**Review by robsdagreat — APPROVED** [review]
Everything looks good but consider the comments, it will be amazing!

**Review by Balinda21 — APPROVED** [review]
Well implemented feature,
great work @Ishimwe7

File: src/pages/seller/NotificationBar.tsx (hunk 2 above)
Comment (Ishimwe7) [reply src/pages/seller/NotificationBar.tsx]: I will remove them

File: src/contexts/notificationContext.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+//   return (
+//     <NotificationContext.Provider value={contextValue}>
+//       {children}
+//     </NotificationContext.Provider>
+//   );
+// };
+
+// export const useNotification = () => {
+//   const context = useContext(NotificationContext);
```
Comment (kirengamartial) [inline src/contexts/notificationContext.tsx]: remove this comment

**Review by kirengamartial — APPROVED** [review]
good @Ishimwe7 and this pr is ready for merge

**Review by derrick-nuby — APPROVED** [review]
The notifications are now real time; which enhances user interaction and a good feebdack loop on the processes you are doing. 

Nice job this PR is ready for merge.

**Pull request merged**